content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -51,106 +65,124 @@ source /etc/profile
 go version
 ```
 ***
-#### Download SODA installer code
+
+### Download SODA Installer code
 ```bash
 git clone https://github.com/sodafoundation/installer.git
 cd installer/ansible
-# Checkout the required version. For example, to checkout v1.3.0 follow
-git checkout v1.3.0
+# Checkout the required version. For example, to checkout v1.4.0 follow
+git checkout v1.4.0
```

Comment:
Please check this PR https://github.com/sodafoundation/documentation/pull/174

Reply:
What should I be looking at?

Reply:
Got it

Reply:
This is handled. The file has been removed from #174